flink-core/src/main/java/org/apache/flink/metrics/groups/AbstractMetricGroup.java

```diff
@@ -100,10 +101,29 @@ public AbstractMetricGroup(MetricRegistry registry, String[] scope) {
 	 * @return fully qualified metric name
      */
 	public String getMetricIdentifier(String metricName) {
+		return getMetricIdentifier(metricName, null);
+	}
+
+	/**
+	 * Returns the fully qualified metric name, for example
+	 * {@code "host-7.taskmanager-2.window_word_count.my-mapper.metricName"}
+	 *
+	 * @param metricName metric name
+	 * @param filter character filter which is applied to the fully qualified metric name
```

Comment:
this is misleading; it is not applied to the fully qualified name (as delimiter's are not filtered)